files/jira.rb

```diff
@@ -4,6 +4,24 @@
 
 class Jira < BaseHandler
 
+  def create_issue_loop(client, issue_json)
+    3.times do
```

Comment:
there is a nice keyword `retry` in ruby: http://blog.mirthlab.com/2012/05/25/cleanly-retrying-blocks-of-code-after-an-exception-in-ruby/

but this is fine too